src/dos/dos_files.cpp

```diff
@@ -1320,6 +1320,23 @@ bool DOS_GetFileDate(Bit16u entry, Bit16u* otime, Bit16u* odate) {
 	return true;
 }
 
+bool DOS_SetFileDate(Bit16u entry, Bit16u ntime, Bit16u ndate) {
+	Bit32u handle=RealHandle(entry);
+	if (handle>=DOS_FILES) {
```

Comment:
Super minor formatting comment. When we add or touch old lines, we try to move to consistent formatting (which also matches with latest code going into upstream too): 
```
	Bit32u handle = RealHandle(entry);
	if (handle >= DOS_FILES) {
```

Reply:
@NicknineTheEagle much better, thank you! :)